ert_gui/simulation/run_dialog.py

```diff
@@ -197,21 +162,45 @@ def sizeHint(self) -> QSize:
     def _setSimpleDialog(self) -> None:
         self._isDetailedDialog = False
         self._tab_widget.setVisible(False)
+        self._job_label.setVisible(False)
+        self._job_view.setVisible(False)
         self.show_details_button.setText("Show Details")
-        self.setFixedHeight(170)
+        self.setFixedHeight(200)
 
     def _setDetailedDialog(self) -> None:
         self._isDetailedDialog = True
         self._tab_widget.setVisible(True)
+        self._job_label.setVisible(True)
+        self._job_view.setVisible(True)
         self.show_details_button.setText("Hide Details")
         self.setFixedHeight(QWIDGETSIZE_MAX)
         self.setMinimumHeight(600)
 
```

Comment:
```suggestion
@Slot(QModelIndex, int, int)
```
It adds _some_ value both socially and technically, to indicate it's a slot.